### DISCUSSION TITLE
cx_freeze and different versions of windows

### DISCUSSION THREAD

**Discussion by durian888** [marcelotduarte/cx_Freeze#1917] [Q&A]
Apologies if this  has been answered previously but my searches failed to turn a suitable answer.

I need to have an application on several generations of window - win 10, 8 and 7  - python 3.8 is usable on these versions of windows (but not on win xp or earlier). I would like to confirm that if a version of cx_Freeze supports a given version of python, that version of cx_Freeze will also support the same range of windows operating systems as the supported version of python? If this not the case, where can I found out which versions of an operating system are supported by a given version of cx_Freeze?

Thank you 

Dominic

**Comment by marcelotduarte** [comment]
As cx_Freeze depends on python, it works on the same OS that python supports. I believe that Python 3.8 supports Win 7 yet, so cx_Freeze will work also. You have to check for other third-party packages also.
[Documentation](https://cx-freeze.readthedocs.io/en/latest/installation.html).
[Versions](https://cx-freeze.readthedocs.io/en/latest/versions.html).
Also, take a look at https://cx-freeze.readthedocs.io/en/latest/faq.html#microsoft-visual-c-redistributable-package
